packages/webcomponents/src/components/payout-transactions-list/payout-transactions-list.tsx

```diff
@@ -0,0 +1,183 @@
+    this.transactionsTable = new Table<PayoutBalanceTransaction>(this.balanceTransactions, this.columns, payoutTransactionTableColumns, payoutTransactionTableCells);
+    checkPkgVersion();
+    this.analytics = new JustifiAnalytics(this);
+    this.initializeApi();
+  }
+
+  disconnectedCallback() {
+    this.analytics.cleanup();
+  }
+
+  @Watch('payoutId')
+  @Watch('authToken')
+  propChanged() {
+    this.initializeApi();
+  }
+
+  private handleError(code: ComponentErrorCodes, errorMessage: string, severity: ComponentErrorSeverity) {
+    this.isLoading = false;
+    this.errorEvent.emit({ errorCode: code, message: errorMessage, severity });
+  }
+
+  private initializeApi() {
+    if (this.payoutId && this.authToken) {
+      const getPayoutTransactions = makeGetPayoutTransactions({
+        authToken: this.authToken,
+        service: new PayoutService(),
+        apiOrigin: this.apiOrigin
+      });
+
+      getPayoutTransactions({
+        params: this.requestParams,
+        onSuccess: ({ balanceTransactions, pagingInfo }) => {
+          this.balanceTransactions = balanceTransactions;
+          this.paging = pagingInfo;
+          this.transactionsTable.collectionData = this.balanceTransactions;
+        },
+        onError: ({ error, code, severity }) => {
+          this.errorMessage = error.message;
+          this.handleError(error, code, severity);
+        },
+        final: () => {
+          this.isLoading = false;
+        }
+      });
+    } else {
+      this.errorMessage = 'payout-id and auth-token props are required';
+      this.handleError(ComponentErrorCodes.MISSING_PROPS, this.errorMessage, ComponentErrorSeverity.ERROR);
+    }
+  }
+
+  handleClickPrevious = (beforeCursor: string) => {
+    this.pagingParams = { before_cursor: beforeCursor };
+    this.clickEvent.emit({ name: TableClickActions.previous });
+  };
+
+  handleClickNext = (afterCursor: string) => {
+    this.pagingParams = { after_cursor: afterCursor };
+    this.clickEvent.emit({ name: TableClickActions.next });
+  };
+
+  rowClickHandler = (e) => {
+    const clickedPayoutId = e.target.closest('tr').dataset.rowEntityId;
+    if (!clickedPayoutId) return;
+
+    const transactionData = this.balanceTransactions.find((payout) => payout.id === clickedPayoutId);
+    this.clickEvent.emit({ name: TableClickActions.row, data: transactionData });
+  };
+
+  get entityId() {
+    return this.balanceTransactions.map((transaction) => transaction.id);
+  }
+
+  get showEmptyState() {
+    return !this.isLoading && !this.errorMessage && this.transactionsTable.rowData.length < 1;
+  }
+
+  get showErrorState() {
+    return !this.isLoading && !!this.errorMessage;
+  }
+
+  get showRowData() {
+    return !this.showEmptyState && !this.showErrorState && !this.isLoading;
+  }
+
+  get requestParams() {
+    return {
+      payout_id: this.payoutId,
+      ...this.pagingParams
+    };
+  }
+
+  render() {
+    return (
+      <StyledHost>
+        <div>
```

Comment:
Is there a need for this extra div?

Reply:
No, good call. This extra div is actually present in all of our `list` components. But it has no styling and doesn't seem to be used for anything. 

I took this opportunity to remove it from all the list components in this PR